packages/rulesets/src/__fixtures__/rules-with-validation-errors.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
+  undeinfed-fn:
```

Comment:
Is this an intentional or accidental typo?

Reply:
Typo :)